portality/background.py

```diff
@@ -79,31 +81,37 @@ def execute(self, background_task):
             job.success()
         job.save()
 
+        # trigger a status change event
+        eventsSvc = DOAJ.eventsService()
+        eventsSvc.trigger(models.Event(constants.BACKGROUND_JOB_FINISHED, job.user.id, {
+            "job": job.id
+        }))
+
         # send a confirmation email to the user if the account exists
         # ~~->Email:Library~~
         # ~~->SearchURLGenerator:Feature
-        if acc is not None:
-            if acc.email is not None and acc.has_role("admin"):
-                template = "email/admin_background_job_finished.jinja2"
-                subject = app.config.get("SERVICE_NAME", "") + " - background job finished"
-
-                url_root = app.config.get("BASE_URL")
-                if not url_root.endswith("/"):
-                    url_root += "/"
-                to = [acc.email]
-                fro = app.config.get('SYSTEM_EMAIL_FROM', 'helpdesk@doaj.org')
-                query = Facetview2.make_query(job.id)
-                url = url_root + "admin/background_jobs?source=" + Facetview2.url_encode_query(query)
-
-                app_email.send_mail(to=to,
-                                    fro=fro,
-                                    subject=subject,
-                                    template_name=template,
-                                    job_id=job.id,
-                                    action=job.action,
-                                    status=job.status,
-                                    background_job_url=url
-                                    )
+        # if acc is not None:
+        #     if acc.email is not None and acc.has_role("admin"):
+        #         template = "email/admin_background_job_finished.jinja2"
+        #         subject = app.config.get("SERVICE_NAME", "") + " - background job finished"
+        #
+        #         url_root = app.config.get("BASE_URL")
+        #         if not url_root.endswith("/"):
+        #             url_root += "/"
+        #         to = [acc.email]
+        #         fro = app.config.get('SYSTEM_EMAIL_FROM', 'helpdesk@doaj.org')
+        #         query = Facetview2.make_query(job.id)
+        #         url = url_root + "admin/background_jobs?source=" + Facetview2.url_encode_query(query)
+        #
+        #         app_email.send_mail(to=to,
+        #                             fro=fro,
+        #                             subject=subject,
+        #                             template_name=template,
+        #                             job_id=job.id,
+        #                             action=job.action,
+        #                             status=job.status,
+        #                             background_job_url=url
+        #                             )
```

Comment:
I think we can just delete this commented out code

Reply:
Removed